h/views/api_flags.py

```diff
@@ -22,6 +22,23 @@ def create(context, request):
     return HTTPNoContent()
 
 
+@api_config(route_name='api.flags',
+            request_method='GET',
+            link_name='flag.index',
+            description='List a users flagged annotations for review.',
+            effective_principals=security.Authenticated)
```

Comment:
Yep, ok. We can probably make this better by distinguishing between 403s that are ok to return to the user and 403s that should be returned as 404s in the API. Probably something that'll be easier to do after we've reintegrated memex.